src/models/StakeRequestRepository.ts

```diff
@@ -159,6 +159,28 @@ export class StakeRequestRepository {
     }
   }
 
+  /**
+   *  Creates stake request models in bulk. This function gracefully
+   *  handles if few records already exists, it only creates new records.
+   *
+   * @param stakeRequestsAttributes Array of stake request attributes.
+   *
+   * @return Array of stake request object.
+   */
+  public async bulkCreate(stakeRequestsAttributes:
+  StakeRequestAttributes[]): Promise<StakeRequest []> {
+    try {
+      const stakeRequests: StakeRequest [] = await StakeRequestModel.bulkCreate(
+        stakeRequestsAttributes,
+        { ignoreDuplicates: true },
```

Comment:
How to handle the duplicates?
 I think that this should be like an `insert or update`, instead of just `insert`.

Reply:
It will ignore duplicate records based on primary key. Only insert non duplicate.

Reply:
So if any other fields other than primary key in the record are different it will not update/insert the record?
I think that we need to create a nonexisting record and update an existing record.

Reply:
I am not sure of this. can you give example of use case?

Reply:
As discussed the entity handler will take care of what is to be updated and inserted.